src/Components/Background/background.jsx

```diff
@@ -0,0 +1,114 @@
+import React, { Component } from 'react';
+import './background.css';
+
+// abstracts circuit
+function circuit(props) {
+  // used for default props
+  const size = props.size || Math.min(window.innerWidth, window.innerHeight);
+  const innerRadius = props.innerRadius || size * 0.125;
+  const outerRadius = props.outerRadius || size * 0.2;
+  const lineSpread = props.lineSpread || 0.4;
+  const endDistance = props.endDistance || 1.85;
+  const lineStartOffset = props.lineStartOffset || 0;
+
+  // placement variables
+  const lineStart = (innerRadius - lineStartOffset ) / Math.sqrt(2);
+  const lineMid = outerRadius / Math.sqrt(2);
+  const lineEnd = outerRadius * endDistance;
+  const spread = (lineSpread * innerRadius) / Math.sqrt(2);
```

Comment:
nit:  not sure what the standard is in js anymore,  but it  might be worth extracting this to a helper function in this file and then returning these values so we can DRY the code up a little bit more

@evanugarte thoughts

Reply:
Great Idea. For now I've extracted them to be global variables instantiated at the top of the file.